### ISSUE TITLE
fdo-client-linuxapp doesn't log ERROR when failing to contact the rendezvous server

### ISSUE THREAD

**Issue by mangelajo** [fedora-iot/fido-device-onboard-rs#201]
The impression was that fdo-client-linuxapp was sitting doing nothing, but I had an incorrect
rendezvous port for the device.

```
 2022-02-09T10:15:41.450Z TRACE hyper::client::pool                   > checkout waiting for idle connection: ("http", fido-rendezvous.ajo.es:8082)
 2022-02-09T10:15:41.450Z DEBUG reqwest::connect                      > starting new connection: http://fido-rendezvous.ajo.es:8082/
 2022-02-09T10:15:41.450Z TRACE hyper::client::connect::http          > Http::connect; scheme=Some("http"), host=Some("fido-rendezvous.ajo.es"), port=Some(Port(8082))
 2022-02-09T10:15:41.450Z DEBUG hyper::client::connect::dns           > resolving host="fido-rendezvous.ajo.es"
 2022-02-09T10:15:41.453Z DEBUG hyper::client::connect::http          > connecting to 10.19.111.203:8082
 2022-02-09T10:15:41.453Z TRACE mio::poll                             > registering event source with poller: token=Token(1), interests=READABLE | WRITABLE
 2022-02-09T10:15:41.615Z TRACE mio::poll                             > deregistering event source from poller
 2022-02-09T10:15:41.615Z TRACE hyper::client::connect::http          > connect error for 10.19.111.203:8082: ConnectError("tcp connect error", Os { code: 111, kind: ConnectionRefused, message: "Connection refused" })
 2022-02-09T10:15:41.615Z TRACE hyper::client::pool                   > checkout dropped for ("http", fido-rendezvous.ajo.es:8082)
 2022-02-09T10:15:41.615Z TRACE fdo_client_linuxapp                   > Error performing TO1 with ServiceClient { protocol_version: Version1_1, base_url: "http://fido-rendezvous.ajo.es:8082", client: Client { accepts: Accepts, proxies: [Proxy(System({}), None)], referer: true, default_headers: {"accept": "*/*"} }, authorization_token: None, encryption_keys: EncryptionKeys { cipher_suite: None, keys: None }, last_message_type: Some(TO1HelloRVAck) }
 2022-02-09T10:15:41.616Z TRACE fdo_client_linuxapp                   > Error Couldn't get TO1 from any Rendezvous server! getting usable To1d from rv_entry RendezvousInterpretedDirective { ip_addresses: None, dns_name: Some("fido-rendezvous.ajo.es"), port: 8082, server_certificate_hash: None, ca_certificate_hash: None, user_input: false, wifi_ssid: None, wifi_password: None, medium: None, protocol: Http, delay: 0, bypass: false }
 2022-02-09T10:15:41.616Z TRACE fdo_client_linuxapp                   > Sleeping for 146 seconds
```

**Comment by 7flying** [comment]
Fixed by #323

**Comment by mangelajo** [comment]
Awesome, thanks!!!
